### ISSUE TITLE
Getting Images from Android photo gallery stopped working from android 10 on?

### ISSUE THREAD

**Issue by hrugani** [codenameone/CodenameOne#3298]
Hi,

We have an app written using codename one that gets images from the photos gallery.
From Android 10 on,  this feature stopped working.

We are using the "Display.getInstance().open" to get the image.

We have observed the image path returned in android 10 have references to files out of the scope of the app itself. i.e: from camera-app, whatsapp, and so on. 

We suppose the feature stopped working for 2 reasons:

1) Codename One could have changed the behavior of the Display class that interacts with the Photos Gallery, returning currently the original image path that is in the File system the belongs to another app.

2) The Android 10 security behavior that blocks the access  

In Summary,
Is there some kind of "Codename one build hint" that could handle that? 

Please, any help on that will be appreciated. 
Our customers too.

**Comment by shai-almog** [comment]
This seems to b be a duplicate of.https://github.com/codenameone/CodenameOne/issues/3204
